.travis.yml

```diff
@@ -1,9 +1,14 @@
 sudo: false
 language: cpp
 compiler:
-- clang
+  - clang
 os:
-- osx
+  - osx
+before_install:
+  - wget https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py
```

Comment:
Perhaps, 
in my head I have uniformity fighting against variable names starting with a capital letter...